doc/manual/src/glossary.md

```diff
@@ -29,6 +51,17 @@
     store object by downloading a pre-built version of the store object
     from some server.
 
+  - [substituter]{#gloss-substituter}\
+    A *substituter* is a store other than `/nix/store` from which Nix will
+    copy a store path instead of building it.  Nix will not copy a store
```

Comment:
```suggestion
    copy store objects it doesn't have. Nix will not copy a store
```
Store path is just a reference, store object is the thing the reference refers to. We've been bad about this in the docs so far, so it's hardly your fault, but we should get better.

Also, it doesn't have to be something we'd otherwise build, it could be something we don't know how to build, that we *have* to substitute.